cli.go

```diff
@@ -208,5 +211,14 @@ func (app *TGFApplication) Run() int {
 		Printf("tgf v%s\n", version)
 		return 0
 	}
+
+	longTimePassed := lastRefresh("auto-update") > 2*time.Hour
```

Comment:
I'd name that `dueForUpdate`

Reply:
Funny, I thought about the SAME EXACT variable name lol 
👍